src/NoteDisplay.cpp

```diff
@@ -425,29 +716,25 @@ struct StripBuffer
 	int Free() const { return size - Used(); }
 };
 
-void NoteDisplay::DrawHoldPart( vector<Sprite*> &vpSpr, int iCol, int fYStep, float fPercentFadeToFail, float fColorScale, bool bGlow,
-				float fDrawDistanceAfterTargetsPixels, float fDrawDistanceBeforeTargetsPixels, float fFadeInPercentOfDrawFar,
-				float fOverlappedTime,
-				float fYTop, float fYBottom,
-				float fYStartPos, float fYEndPos,
-				bool bWrapping, bool bAnchorToTop, bool bFlipTextureVertically )
+void NoteDisplay::DrawHoldPart(vector<Sprite*> &vpSpr,
```

Comment:
I appreciate the parameters being brought in like this. However, I wonder if we're passing too many.

Would it make more sense to pass in a HoldPart object?